docs/90_partners.md

```diff
@@ -1,17 +1,36 @@
 # Project partners
 
-* Walther Blom, Dell Technologies, High Education & Research
-* Jaco van Dijk, Dell Technologies, Higher Education
-* Bob Dröge, Center for Information Technology, University of Groningen
-* Robbert Eggermont, Delft University of Technology
-* Kenneth Hoste, HPC-UGent
-* Geert Jan Laanstra, Electrical Engineering, Mathematics and Computer Science (EEMCS), University of Twente, Enschede
-* Caspar van Leeuwen, SURF
-* Koen Mulderij, Delft University of Technology
-* Alan O'Cais, Juelich Supercomputing Centre
-* Thomas Röblitz, University of Oslo
-* Mark Sharpley, University of Cambridge Research Computing Services Division
-* Peter Stol, Vrije Universiteit Amsterdam
-* Marco Verdicchio, SURF
-* Bas van der Vlies, SURF
-* Henk-Jan Zilverberg, Center for Information Technology, University of Groningen
+## Delft University of Technology
+* Robbert Eggermont
+* Koen Mulderij
+
+## Dell Technologies
+* Walther Blom, High Education & Research
+* Jaco van Dijk, Higher Education
+
+## HPC-UGent
+* Kenneth Hoste
+
+## J&uuml;lich Supercomputing Centre
```

Comment:
I think `Jülich` is fine?

Reply:
`Juelich` is also common/accepted when the umlaut is not a convenient option.

Reply:
@boegel That appears to work, I used the &... notation, since that's the official html way to do it. However your way is easier to read when editing.. I will change it.